source/annotation.html

```diff
@@ -6,20 +6,19 @@
 	<link rel="stylesheet" href="styles/annotation.css">
     <body>
 		<div class="container container-table">
+			<div id="bad"></div>
 			<div class="row vertical-center-row">
 				<div class="polaroid">
 					<div class="polaroid-image">
-						<img id="image" src=""></img>
+						<img id="image" src="">
```

Comment:
Actually, it doesn't need a closing tag unless it has something within the two tags 👌